clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiAdaptor.groovy

```diff
@@ -60,6 +62,58 @@ class KubernetesApiAdaptor {
     client.replicationControllers().inNamespace(namespace).withName(name).scale(size)
   }
 
+  void removePodLabels(String namespace, String name, List<String> keys) {
+    def edit = client.pods().inNamespace(namespace).withName(name).edit().editMetadata()
+
+    keys.forEach {
```

Comment:
Any specific reason to use the java8 `.forEach` here instead of groovy's `.each`? I noticed you used `.each` down below in `KubernetesUtil. getPodLoadBalancers()`.